factory/interface.go

```diff
@@ -610,6 +611,7 @@ type RunTypeComponentsHolder interface {
 	ShardResolversContainerFactoryCreator() resolverscontainer.ShardResolversContainerFactoryCreator
 	TxPreProcessorCreator() preprocess.TxPreProcessorCreator
 	ExtraHeaderSigVerifierHandler() headerCheck.ExtraHeaderSigVerifierHolder
+	GenesisBlockCreator() processComp.GenesisBlockCreatorFactory
```

Comment:
Maybe?

```suggestion
	GenesisBlockCreatorFactory() processComp.GenesisBlockCreatorFactory
```

Or Rename the factory interface to simply `Creator` ?

Reply:
renamed to `GenesisBlockCreatorFactory()`